[[patch.notes.version.4.0]]
= Documentation Patch Notes for {docversion}
include::ROOT:partial$entities.adoc[]
:revnumber: V4.0
:revdate: February 21, 2019



== What's New?

{revnumber} {revdate}



=== Information Architecture Rework

We have received a large amount of feedback and therefore our docs are undergoing an information architecture rework.

== Known Issues

. Cross references between docs are being passed during our build process to prevent errors and therefore may not work as expected. Throughout documentation you will find the following syntax listed in place of our normal links:
+

====
pass:c[xref:FILENAME.adoc#SECTION-ID[]]
====
+

As we continue working the correct references will be added and the legacy ID's will be removed.

. During the IA process we discovered several missing topics that you may find useful. These topics have not yet been written and will show up as empty when selected. These will be addressed once current content has been properly placed within the new framework.



== Patch Notes for {productname} {productnumber}

Coming Soon...